kinto/api-test/mockclient.py

```diff
@@ -40,6 +39,13 @@ def post_request(self, resource, data=None, status_code=None):
             r.raise_for_status()
         return json.loads(r.content.decode('utf-8'))
 
+    def post_json_request(self, resource, data=None, status_code=None):
```

Comment:
does this need to be a separate request type? maybe we can use the json.dumps method as in http://docs.python-requests.org/en/latest/user/quickstart/#more-complicated-post-requests